.github/workflows/ci.yml

```
@@ -1,32 +1,53 @@
 ## This is the name of the CI
 name: CI
 
-## These are the actions that trigger the CI to run
-on: [push, pull_request]
+## These are the triggers for the CI to run
+on: [ push , pull_request ]
 
 jobs:
+
   build:
-    # TODO: Eventually include different OS's in the matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        # We build on our supported version of coq and the master version
+        coq_version: [ '8.13' , 'latest' ]
+        ocaml_version: [ '4.11-flambda' ]
+        
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout branch
+    - uses: actions/checkout@v2
+    # Checkout submodules
+    - uses: snickerbockers/submodules-init@v4
+    # We use the coq docker so we don't have to build coq
+    - uses: coq-community/docker-coq-action@v1
+      with:
+        coq_version: ${{ matrix.coq_version }}
+        ocaml_version: ${{ matrix.ocaml_version }}
+        custom_script: |
+          startGroup "Workaround permission issue" # https://github.com/coq-community/docker-coq-action#permissions
+          sudo chown -R coq:coq .
+          endGroup
+          etc/coq-scripts/github/reportify-coq.sh --errors make TIMED=1 -j2     --output-sync GH_REPORT_ERRORS=1
```

We should also pass `--warnings` iff we're building for Coq master.  And I think we don't want `GH_REPORT_ERRORS=1`
```suggestion
          etc/coq-scripts/github/reportify-coq.sh --stdout --errors ${{ matrix.extra_gh_reportify }} etc/coq-scripts/timing/make-pretty-timed.sh -j2 TIMING=1
          make
```